draft-ietf-rats-corim.md

```diff
@@ -539,6 +541,41 @@ Described in {{sec-common-validity}}.
 {::include cddl/unprotected-corim-header-map.cddl}
 ~~~
 
+## Signer authority of securely conveyed unsigned CoRIM {#sec-conveyed-signer}
+
+An unsigned (#6.501-tagged) CoRIM may be a payload in an enveloping signed document.
+An unsigned CoRIM may be a payload within a secure channel.
+The CoRIM signer authority is taken from the authenticated credential of the same entity that originates the CoRIM.
+A CoRIM role entry that contains the `manifest-signer` role MUST be added to `corim-entity-map`.
+
+It is out of scope of this document to specify a method of delegating the signer role in the case that an unsigned CoRIM is conveyed through multiple secured links with different notions of authenticity without end-to-end integrity protection.
+
+### CoRIM bundles
+
+A method of signing a bundle of CoRIMs together is through a signed RATS Conceptual Message Wrapper (CMW) {{-cmw}}.
+The COSE_Sign1 signature format can be used with a Collection CMW.
+The COSE protected header can include a CMW collection type name.
+The collection type name SHALL be of the form: `tag:{{&SELF}}:bundle`.
```

Comment:
We should use the terminology of CMW Collection Profile

Reply:
/cc @thomas-fossati